sentry/src/main/java/io/sentry/HubScopesWrapper.java

```diff
@@ -169,8 +169,8 @@ public void withScope(@NotNull ScopeCallback callback) {
   }
 
   @Override
-  public void configureScope(@NotNull ScopeCallback callback) {
-    scopes.configureScope(callback);
+  public void configureScope(@Nullable ScopeType scopeType, @NotNull ScopeCallback callback) {
```

Comment:
`m`: I guess we should do the same for `withScope`

Reply:
Yeah, will add in a follow up PR.

Reply:
Ah, this has the same problem as having an overload for `forkedScope` where `withScope(ISOLATION) ...` forks both isolation and current scope which might be confusing.

Maybe having another method called `withIsolationScope`, as RFC suggests, is better but very similar.